src/files.ts

```diff
@@ -462,3 +461,22 @@ export function* composeFilePayloads(files: FileInfo[], bucketSize = MAX_PAYLOAD
 export function isMatch(filePath: string, rules: string[]): boolean {
   return !!multimatch([filePath], rules, { ...multiMatchOptions, matchBase: false }).length;
 }
+
+export function parseDotSnykExcludes(pathToDotSnykFile: string): string[] {
+  try {
+    const dotSnykFile = fs.readFileSync(pathToDotSnykFile, 'utf-8');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const parsed: { exclude: string[] } = parseYaml(dotSnykFile);
+    return parsed.exclude.map(path => `${nodePath.dirname(pathToDotSnykFile)}/${path}`);
+  } catch (err) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    if (err.code === 'EACCES' || err.code === 'EPERM') {
+      console.error(`${pathToDotSnykFile} is not accessible.`);
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    if (err.code === 'ENOENT') {
+      console.error(`no such file or directory: ${pathToDotSnykFile}`);
```

Comment:
How is this an error? Many directories won't have a `.snyk` file, at most it's an info level log.